### PR TITLE
Bash scripts used for backups

### PR DESCRIPTION
**Pull request by huubnijs**
These all are the scripts for production. Test-servers have the same scripts, but different configurations.

### CODE REVIEW

File: scripts/dashboardbackup.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This script is used on prod-salt
+# It uses wizzy to read the Grafana-dashboards and dump them
+#     utkarshcmu.github.io/wizzy-site/home
```
Comment (mjs) [inline scripts/dashboardbackup.sh]: Is this correct?

File: scripts/pgsql_backup.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+MC=/usr/local/bin/mc
+CURL=/usr/bin/curl
+HOST=$(grep " hostname =" /etc/telegraf/telegraf.conf |cut -d"\"" -f2)
+DATABASE=cacodb02
+LOCALSTORE=/mnt/database-backup
+REMOTESTORE=cat-nz-por-1/cacophony-backup/postgresql
+DATESTAMP=`date  +%F`
+DAYS=30
+MAILTO=huubnijs@gmail.com
```
Comment (mjs) [inline scripts/pgsql_backup.sh]: Can we send to coredev@cacophony.org.nz as well?

File: scripts/dashboardbackup.sh (hunk 1 above)
Comment (huubnijs) [reply scripts/dashboardbackup.sh]: Yes, I have made it clear now that that is a URL for documentation

File: scripts/pgsql_backup.sh (hunk 2 above)
Comment (huubnijs) [reply scripts/pgsql_backup.sh]: Replaced the mail-address

**Comment by huubnijs** [comment]
I have added mail-after-failure in all 3 scripts. And I hope it is more readable now.

**Pull request merged**